Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.cpp

```diff
@@ -56,16 +56,16 @@ namespace ProcessContainers {
             };
 
             Core::File configBundleFile(path + "/Container" + CONFIG_NAME);
-            TRACE(ProcessContainers::ProcessContainerization, (_T("searching %s container at %s, %s"), id.c_str(), configBundleFile.Name().c_str(), configBundleFile.PathName().c_str()));
```

Comment:
Why did you change the trace category? Using the ProcessContainers category enables easily filtering on container messages. Might it not compile then let's fix that but not change the category